src/ast/variables/LocalVariable.ts

```diff
@@ -18,7 +15,7 @@ export default class LocalVariable extends Variable {
 
 	constructor (
 		name: string, declarator: Identifier | ExportDefaultDeclaration | null,
-		init: Expression | Declaration | UnknownAssignment
+		init: Expression
```

Comment:
Are you sure this will always be an expression? I may be wrong, but I seem to remember testing this before and seeing declarations come up.

Reply:
Well, the declarations in question all satisfy the `Expression` interface now (and `Declaration` no longer is a concept). This will probably be clearer once I employ your interface naming suggestions.